runtime/gc_base/modronapi.cpp

```diff
@@ -1145,11 +1147,14 @@ j9gc_reinitialize_for_restore(J9VMThread *vmThread, const char **nlsMsgFormat)
 	}
 	acquireVMAccess(vmThread);
 
-	if (NULL != verboseGCManager) {
-		/* Verbose gc reinit failure can be ignored, it is safe for restore to continue on. */
-		verboseGCManager->reinitializeForRestore(env);
+	if (!mmFuncTable->checkOptsAndInitVerbosegclog(vm, vm->checkpointState.restoreArgsList)) {
```

Comment:
keep in mind that `vm->checkpointState.restoreArgsList` can be NULL

Reply:
Would that be an issue if we pass NULL through to something like 

` FIND_AND_CONSUME_ARG(args, OPTIONAL_LIST_MATCH, OPT_XVERBOSEGCLOG, NULL);`

Would this crash or simply return as though the check opt is not found?

Reply:
```C++
checkOptsAndInitVerbosegclog(J9JavaVM* vm, J9VMInitArgs* args)
{
	IDATA verbosegclogIndex = FIND_AND_CONSUME_ARG(args, OPTIONAL_LIST_MATCH, OPT_XVERBOSEGCLOG, NULL);

	if (verbosegclogIndex >= 0) {
		if (!initializeVerbosegclog(vm, verbosegclogIndex, args)) {
```

My assumption is that, given null args, verbosegclogIndex will fail the check

Reply:
I recently added a NULL check to `findArgInVMArgs` so `FIND_AND_CONSUME_ARG` is fine. But other operations may be assuming non-NULL since `vm->vmArgsArray` has always been non-NULL